enumeratum-core/src/test/scala/enumeratum/EnumSpec.scala

```diff
@@ -250,7 +250,28 @@ class EnumSpec extends FunSpec with Matchers {
         case object Bar extends Foo
         case object Baz extends Foo
       }
-        """ shouldNot compile
+      """ shouldNot compile
+    }
+  }
+
+  describe("in") {
+    import DummyEnum._
+
+    it("should return true if enum value is contained by the parameter list") {
+      val enum: DummyEnum = Hello
```

Comment:
Yes it is necessary, because the scala compiler is quite smart :) If you don't specify the type of "val enum:DummyEnum", then the type will be Hello.type, and the "in" function will only accept this type. But this is not a realistic scenario, in a real world situation the type of the variable will be eg. DummyEnum, not the concrete EnumValue.